src/AspNetCore.Identity.MongoDB/MongoIdentityUser.cs

```diff
@@ -33,7 +33,7 @@ public MongoIdentityUser(string userName)
                 throw new ArgumentNullException(nameof(userName));
             }
 
-            Id = GenerateId(userName);
+            // Id = GenerateId(userName); // Id will generate by mongodb driver
```

Comment:
we can delete this comment and `GenerateId` impl. I believe `GenerateId` is not used anymore?